typings/index.d.ts

```diff
@@ -1632,6 +1627,8 @@ export class MessageActionRow<
     ? APIActionRowComponent<APIModalActionRowComponent>
     : APIActionRowComponent<APIMessageActionRowComponent>,
 > extends BaseMessageComponent {
+  // @ts-expect-error (TS:2344, Caused by TypeScript 4.8)
```

Comment:
```suggestion
  // @ts-ignore (TS:2344, Caused by TypeScript 4.8)
```

Am on work laptop w/o the repo cloned, this is faster to process @imranbarbhuiya's comment.